### PR TITLE
[ABW-2559] No more relay

### PR DESCRIPTION
**Pull request by kugel3**
## Ticket
[ABW-2559](https://radixdlt.atlassian.net/browse/ABW-2559)

## Description
-Remove Relay from TransferAccountList
-Remove Relay from DappInteractionFlow
-Remove Relay from DappInteractor
-Remove usage of special scope in OneTimePersonaData
-Upgrade to (stock version of) TCA 1.7

## Note
It is almost possible to use a pretty nice automatic solution that replaces the need for relays in DappInteractionFlow (see the code that's deleted in the last commit).

## How to test

Especially test:
- Selecting accounts and assets in asset transfer
- Various dApp interaction flows
- One time persona data requests

## PR submission checklist
- [x] I have tested account to account transfer flow and have confirmed that it works

[ABW-2559]: https://radixdlt.atlassian.net/browse/ABW-2559?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -185,8 +185,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/radixdlt/swift-composable-architecture",
       "state" : {
-        "branch" : "full-scope-1.4.2",
-        "revision" : "789fab0d836fb230b5e279f4a62d9112e4d6d81e"
+        "revision" : "30ab89daa4e392ecd2150f737c42dbb2336bbc04",
+        "version" : "1.7.0"
```
Comment (GhenadieVP) [inline RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:189]: Not resolved to latest 1.7.3?

**Review by GhenadieVP — APPROVED** [review]
Long awaited change 💯 💪!
Will perform also some local testing for the impacted flows.

File: RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (kugel3) [reply RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:189]: Actually not so strange, it was still using our fork, but the main branch there. Now I switched to the official repo.

File: RadixWallet/Features/AccountAndPersonaHidingFeature/AccountAndPersonaHiding+Reducer.swift [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+@Reducer
 public struct AccountAndPersonaHiding: Sendable, FeatureReducer {
+	// MARK: - State
+
 	public struct State: Hashable, Sendable {
 		public var hiddenEntityCounts: EntitiesVisibilityClient.HiddenEntityCounts?
 
 		@PresentationState
-		public var confirmUnhideAllAlert: AlertState<ViewAction.ConfirmUnhideAllAlert>?
+		public var destination: Destination.State? = nil
```
[the remark this answers is not in the document]
Comment (kugel3) [reply RadixWallet/Features/AccountAndPersonaHidingFeature/AccountAndPersonaHiding+Reducer.swift:10]: Yes, but that is part of the Observation changes it seems. We first need to migrate to CasePathable...

**Review by CyonAlexRDX — APPROVED** [review]
Nice to not use a fork anymore 🥳

**Pull request merged**